Items/06_Explosives.txt

```diff
@@ -70,16 +73,18 @@ investigation checks.
 == ECSC Fragmentation Grenade $90 ==
 
 Damage: D10 Thermal Damage to exact spot of impact
-Ranged Miss Chance (to land where intended): 5 out to 10+[DEX or STR]*2 meters
+Miss Chance: 5
+Range: 10+[DEX or STR]*2 meters
 Armor Piercing Level: 1
 Reflex Modifier: -10
 Reflex Save: 100
 
-    Each Target within the blast radius takes multiple hits. 45 ballistic 
+    Each Target within the blast radius takes three shrapnel hits. 45 ballistic 
 damage per hit, automatically hits armor. Targets take full damage to the torso, 
 legs, and arms (if armor is pierced or not present): (3*45 damage)+2xBleed 
 within a 2.5m radius.Only legs and torso (2*45 damage)+1xBleed within a 5m 
```

Comment:
Another thing for this frag: I'd lead with "If all three shrapnel hits pierce the target's armor and the target is within 2.5m of the explosion, the target takes wounds to the torso, arms, and legs for a total of:.... Within a 5m radius of the explosion, targets take two shrapnel hits. If both hits pierce the target;s armor, they take wounds to the legs and torso for a total of:.... If only one shrapnel hit pierces the target's armor, they take a single wound to their torso for a total of:.... Targets do not suffer any Bleed effect unless at least one hit deals damage to their health.